libs/traits/src/lib.rs

```diff
@@ -539,6 +539,23 @@ impl<Amount, Currency: PartialEq> Swap<Amount, Currency> {
 	}
 }
 
+/// The information of a swap order
+#[derive(Clone, Encode, Decode, Eq, PartialEq, RuntimeDebug, TypeInfo, MaxEncodedLen)]
+pub struct OrderInfo<Balance, Currency, Ratio> {
+	/// The underlying currency swap
+	pub swap: Swap<Balance, Currency>,
+	/// The ratio at which the swap should happen
+	pub ratio: OrderRatio<Ratio>,
+}
+
+// impl<Balance, Currency, Ratio> From<OrderInfo<Balance, Currency, Ratio>>
+// 	for Swap<Balance, Currency>
+// {
+// 	fn from(&self) -> Swap<Balance, Currency> {
+// 		self.swap
+// 	}
+// }
+
```

Comment:
NIT: Could be removed if not needed?

Reply:
Good catch! I thought I had removed it but couldn't find it anymore.